### PR TITLE
Update summary-preview1-java with EventHubs information

### PR DESCRIPTION
**Pull request by conniey**
* Add information and links to Event Hubs packages/readme.

@loarabia: I wasn't sure of the format you wanted for the Changelog items (ie. How to separate issues/features between each client library). 
Also, not sure where our preview documentation is at.

See: https://github.com/Azure/azure-sdk-for-java/pull/4244 for CHANGELOG.md PR.

### CODE REVIEW

**Comment by JonathanGiles** [comment]
I'm happy to do a pass on this doc once this PR is merged. Question for @loarabia - should this be renamed to no longer reference 'preview 1', seeing as by the time we do this announcement, some of our libraries will be preview 2 or greater? Do we refresh this to July?

File: docs/release-notes/preview/summary-preview1-java.md [hunk 1]
```diff
@@ -1,20 +1,25 @@
-# Azure SDK for Java Preview 1 Release Notes 
+# Azure SDK for Java Preview 1 Release Notes
```
Comment (adrianhall) [inline docs/release-notes/preview/summary-preview1-java.md]: Suggest "Azure SDK for Java (July 2019 Preview) Release Notes"

File: docs/release-notes/preview/summary-preview1-java.md [hunk 2]
```diff
@@ -1,20 +1,25 @@
...
 
 TODO High level info about the release what goals, what libraries are coming with this release, key new features.
```
Comment (adrianhall) [inline docs/release-notes/preview/summary-preview1-java.md]: Can you copy the initial paragraph from the blog post, with a "For more details, see [the blog post]" - we can add the latter after July 9th

File: docs/release-notes/preview/summary-preview1-java.md [hunk 3]
```diff
@@ -1,20 +1,25 @@
...
+  <artifactId>azure-messaging-eventhubs</artifactId>
+  <version>5.0.0-preview.1</version>
+</dependency>
+
+<!-- TODO add the rest -->
 ```
 
-
 ## Feedback
 If you have a bug or feature request for one of the libraries, please post an issue to TODO [GitHub](https://github.com/azure/azure-sdk-for-java/issues) and add the labels Preview and Bug or Feature
```
Comment (adrianhall) [inline docs/release-notes/preview/summary-preview1-java.md]: Need to remove the TODO

File: docs/release-notes/preview/summary-preview1-java.md [hunk 4]
```diff
@@ -28,4 +33,5 @@ TODO: Replace with the top 5 to 10 Features, Known Issues, or Migration pointers
 TODO: fill out table and update links
 | Service  | Install | Quickstart |  API Reference | Changelog |
 | -- | -- | -- | -- | -- |
-| Storage Blobs | TODO [package](https://crates.io/crates/azure_sdk_for_rust) | TODO [Readme.md](github.com) | TODO [Preview Documentation](azure.github.io) | TODO [Changelog.md](github.com) |
\ No newline at end of file
+| Storage Blobs | TODO [package](https://crates.io/crates/azure_sdk_for_rust) | TODO [Readme.md](github.com) | TODO [Preview Documentation](azure.github.io) | TODO [Changelog.md](github.com) |
```
Comment (adrianhall) [inline docs/release-notes/preview/summary-preview1-java.md]: Errr - rust????

File: docs/release-notes/preview/summary-preview1-java.md (hunk 3 above)
Comment (joshfree) [reply docs/release-notes/preview/summary-preview1-java.md]: yes, also please quote Preview, Bug, Feature

**Comment by conniey** [comment]
> Are we only releasing two SDKs?
@adrianhall No we aren't. The goal of these changes was to fill in items that pertain to Event Hubs and let all the other feature owners fill in their own items.

**Comment by JonathanGiles** [comment]
I'm merging this PR so that I may take a turn on making changes.

**Comment by JonathanGiles** [comment]
Also, I will take on the feedback proposed in this PR and make appropriate changes in a follow-up PR.

File: docs/release-notes/preview/summary-preview1-java.md (hunk 1 above)
Comment (JonathanGiles) [reply docs/release-notes/preview/summary-preview1-java.md]: Done

File: docs/release-notes/preview/summary-preview1-java.md (hunk 2 above)
Comment (JonathanGiles) [reply docs/release-notes/preview/summary-preview1-java.md]: @adrianhall, are you able to get me this paragraph? For now I've just written my own text, but I can replace this once I get the new text.

**Pull request merged**